### ISSUE TITLE
API Changes

### ISSUE THREAD

**Issue by WWOL** [madleech/FlickElectricApi#2]
Recently Flick has changed their API a bit. There is no longer a "commentary" part and some new things have been added. The response I get to the `customer/mobile_provider/price` endpoint now looks like this;
```json
 {
    "kind":"mobile_provider_price",
    "customer_state":"active",
    "needle":{
        "price":"13.347",
        "unit_code":"cents",
        "per":"kwh",
        "start_at":"2017-05-06T08:30:00Z",
        "end_at":"2017-05-06T08:59:59Z",
        "now":"2017-05-06T08:42:24.092Z",
        "type":"rated",
        "charge_methods":["kwh","spot_price"],
        "components":[{"charge_method":"kwh","value":"0.113"},{"charge_method":"kwh","value":"3.5"},{"charge_method":"kwh","value":"3.625"},{"charge_method":"spot_price","value":"6.109"}]
    }
}
```

I don't actually use this library (so not sure if this breaks it), but your research got me started when doing my own thing, so thanks.

**Comment by madleech** [comment]
Thanks for the heads up. In the process of moving house so will take a look when I've moved.

**Comment by madleech** [comment]
Fixed in #5.
